src/EGL/icd_enumeration.md

```diff
@@ -0,0 +1,113 @@
+        might require loader changes.
+    * The key `ICD` must have an object value.
+        * In the `ICD` object, the key `library_path` must have a string value.
+            * If the library path is a bare filename with no directory
+                separators, for example `libEGL_mesa.so.0`, the loader is
+                expected to search for it in the system's default runtime
+                library directories.
+            * If the library path is an absolute path, for example
+                `/opt/myvendor/libEGL_myvendor.so`, the loader is expected
+                to load the ICD from that absolute path.
+            * If the library path is a relative path containing at least
+                one directory separator, for example `./libEGL_myvendor.so`,
+                the loader's behaviour is currently unspecified.
+
+## ICD installation
+
+The JSON files describing ICDs can be installed into
+`${datadir}/glvnd/egl_vendor.d` or `${sysconfdir}/glvnd/egl_vendor.d`,
+where `${datadir}` and `${sysconfdir}` are the data and system
+configuration directories with which which GLVND was configured,
+normally `/usr/share` and `/etc` respectively. These paths can be
+discovered by using pkg-config.
+
+ICDs installed into the same directory hierarchy (prefix) as GLVND itself
+should normally have their JSON files installed into the `${datadir}`.
+
+If a vendor installs ICD libraries into one of the system's default
+runtime library directories, they must install a JSON file referencing
+the ICD library by either its absolute path or its bare filename.
+
+If a vendor installs ICD libraries into a non-standard directory, they
+must install a JSON file referencing the ICD library by its absolute path.
+
+If a vendor installs matching ICD libraries for more than one ABI
+(for example 32- and 64-bit versions), each library has the same name,
+and each library is in a directory searched by default by the runtime
+linker and referenced by its bare filename, then they may install a
+single JSON file to describe all of those ICD libraries. For example,
+Mesa on Debian installs `/usr/lib/x86_64-linux-gnu/libEGL_mesa.so.0` and
+`/usr/lib/i386-linux-gnu/libEGL_mesa.so.0`, which can both be described
+by a JSON file with `"library_path": "libEGL_mesa.so.0"`. Similarly, Mesa
+on Arch Linux installs 64-bit `/usr/lib/libEGL_mesa.so.0` and 32-bit
+`/usr/lib32/libEGL_mesa.so.0`, with a similar JSON file, while some
+other distributions use `lib64` for 64-bit libraries.
+
+## Examples
+
+On systems with the Mesa EGL drivers, this file is typically installed at
+`/usr/share/glvnd/egl_vendor.d/50_mesa.json`:
+
+```
+{
+    "file_format_version" : "1.0.0",
+    "ICD" : {
+        "library_path" : "libEGL_mesa.so.0"
+    }
+}
+```
+
+A third-party ICD installed at `/opt/myvendor/lib64/libEGL_myvendor.so`
+could install this file in `/etc/glvnd/egl_vendor.d/10_myvendor.x86_64.json`,
+or install it in
```

Comment:
I don't think a symlink bears special mention here. Libglvnd doesn't care if it's a symlink or a regular file, but calling it out suggests that there's some special treatment of symlinks.

Reply:
I was trying to leave a hint that installation of ICDs into non-system prefixes like `/opt` maybe *should* consider using symlinks :-)

But OK, I'll remove that.

Reply:
Done